src/check/models.rs

```diff
@@ -164,6 +207,25 @@ pub(crate) fn check_model(
         Err(failure) => failures.push(failure),
     }
 
+    match enforce_model_properties_layout(&mut working_model, config) {
+        Ok(Some(change)) => {
+            if let ModelChange::NormalizePropertiesLayout {
```

Comment:
I think it's fine but technically possible that enforce_model_properties_layout could give a different ModelChange?

Probably fine.